index.html

```diff
@@ -477,7 +477,31 @@ <h3>Required Owned Elements</h3>
 		</section>
 		<section id="scope">
 			<h3>Required Context Role</h3>
-			<p>The required context role defines the owning container where this <a>role</a> is allowed. If a role has a required context, authors MUST ensure that an element with the role is contained inside (or <a>owned</a> by) an element with the required context role. For example, an element with role <code>listitem</code> is only meaningful when contained inside (or owned by) an element with role <code>list</code>.</p>
+			<p>The required context role defines the owning container where this <a>role</a> is allowed. If a role has a required context, authors MUST ensure that the element is contained inside (or <a>owned</a> by) an element with the required context role. There can be no other intermixed containing elements unless those elements are explicitly set as presentational, e.g. with a <rref>none</rref> role.</p>
+			<p>For example, an element with role <code>listitem</code> is only meaningful when contained inside (or owned by) an element with role <code>list</code>, as demonstrated by the following code snippet.</p>
+			<div class="example">
```

Comment:
Needs to be a `<pre>` or an `<aside>` in order to get all the example text from respec. See https://github.com/w3c/respec/wiki/example

Reply:
Fixed